application/views/templates/footer.php

```diff
@@ -1,2 +1,11 @@
+<!-- Global site tag (gtag.js) - Google Analytics -->
+<script async src="https://www.googletagmanager.com/gtag/js?id=UA-2753771-46"></script>
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){dataLayer.push(arguments);}
+  gtag('js', new Date());
+
+  gtag('config', 'UA-2753771-46');
```

Comment:
make it configurable.

Reply:
If I want to make it configurable, that means another environment variable.